spark/src/main/scala/geotrellis/spark/io/accumulo/AccumuloLayerUpdater.scala

```diff
@@ -17,25 +17,29 @@ class AccumuloLayerUpdater[K: Boundable: JsonFormat: ClassTag, V: ClassTag, Cont
   extends LayerUpdater[LayerId, K, V, Container with RDD[(K, V)]] {
 
   def update(id: LayerId, rdd: Container with RDD[(K, V)]) = {
-    try {
-      if (!attributeStore.layerExists(id)) throw new LayerNotExistsError(id)
-      implicit val sc = rdd.sparkContext
-      implicit val mdFormat = cons.metaDataFormat
+    if (!attributeStore.layerExists(id)) throw new LayerNotFoundError(id)
+    if (rdd.isEmpty()) throw new LayerUpdateError(id, ": empty rdd update")
```

Comment:
This actually computes a partition, is there a way to check this that doesn't perform an action?